src/libstore/build/local-derivation-goal.cc

```diff
@@ -1666,6 +1666,7 @@ void LocalDerivationGoal::runChild()
                 ss.push_back("/dev/null");
                 ss.push_back("/dev/random");
                 ss.push_back("/dev/tty");
+                ss.push_back("/dev/fuse");
```

Comment:
Note that this causes a build failure if `/dev/fuse` doesn't exist on the host filesystem.